components/script/dom/xrsession.rs

```diff
@@ -256,7 +257,7 @@ impl XRSession {
     //
     // This enables content that assumes all input sources are accompanied
     // by an inputsourceschange event to work properly. Without
-    pub fn setup_initial_inputs(&self) {
+    pub fn setup_initial_inputs(&self, can_gc: CanGc) {
```

Comment:
This one is only used inside of an async task, so let's remove it and use CanGc::note() instead.